src/index.html

```diff
@@ -17,6 +30,87 @@
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <h1>MOYO HEADER</h1>
+    <header>
+      <a
+        class="logo"
+        href="#logo"
+      >
+        <img
+          src="./images/logo.png"
+          alt="logo MOYO"
+        />
+      </a>
+
+      <nav>
+        <ul>
+          <li>
+            <a
+              href="#apple"
+              class="link is-active first-box"
```

Comment:
use pseudo-classes instead of adding additional classes (first-box and last-box)